Python/Product/PythonTools/PythonToolsService.cs

```diff
@@ -579,13 +561,16 @@ public sealed class PythonToolsService : IDisposable {
 
         internal System.IServiceProvider Site => _container;
 
-        internal LanguagePreferences LangPrefs {
-            get {
-                return _langPrefs;
+        internal LanguagePreferences LangPrefs => _langPrefs.Value;
+
+        internal async Task<LanguagePreferences> GetLangPrefsAsync() {
```

Comment:
I think that deserves a comment.  If I need preferences, how do I know if I'm supposed to call the method or the propery?